atd-vzv/src/views/map/InfoBox/MapInfoBox.js

```diff
@@ -29,7 +29,7 @@ const MapInfoBox = React.memo(
     const buildSeriousInjuriesOrFatalitiesConfig = (info) => [
       {
         title: "Date/Time",
-        content: moment(info.crash_date).format("MM/DD/YYYY HH:MM A"),
+        content: moment(info.crash_date).format("MM/DD/YYYY HH:mm A"),
```

Comment:
niiice catch! fwiw—if you wanted to move this over to `date-fns` i think it might be the same tokens?

Reply:
yeah it would be a good idea to create a spin off issue to replace moment with date-fns in the entire VZV codebase bc looks like its used in lots of diff places. probably should have just addressed vzv when i did this for [vze](https://github.com/cityofaustin/atd-vz-data/pull/1114) 😅

Reply:
Good call, y'all! Just created https://github.com/cityofaustin/atd-data-tech/issues/12007 to cover this. cc @patrickm02L